### PR TITLE
updated for xcode 8 GM

### PR DESCRIPTION
**Pull request by migfabio**

### CODE REVIEW

File: TagListView/TagListView.swift [hunk 1]
```diff
@@ -189,35 +189,35 @@ public class TagListView: UIView {
...
     
-    @IBOutlet public weak var delegate: TagListViewDelegate?
+    @IBOutlet open weak var delegate: TagListViewDelegate?
     
-    public private(set) var tagViews: [TagView] = []
-    private(set) var tagBackgroundViews: [UIView] = []
-    private(set) var rowViews: [UIView] = []
-    private(set) var tagViewHeight: CGFloat = 0
-    private(set) var rows = 0 {
+    open fileprivate(set) var tagViews: [TagView] = []
```
Comment (xhacker) [inline TagListView/TagListView.swift]: I’d keep this `private` instead of `fileprivate`.

File: TagListView/TagListView.swift [hunk 2]
```diff
@@ -189,35 +189,35 @@ public class TagListView: UIView {
...
     
-    public override func layoutSubviews() {
+    open override func layoutSubviews() {
         super.layoutSubviews()
         
         rearrangeViews()
     }
     
-    private func rearrangeViews() {
+    fileprivate func rearrangeViews() {
```
Comment (xhacker) [inline TagListView/TagListView.swift]: Also, this should be `private`, right?

File: TagListView/TagListView.swift (hunk 1 above)
Comment (migfabio) [inline TagListView/TagListView.swift]: 👍

File: TagListView/TagView.swift [hunk 3]
```diff
@@ -162,7 +162,7 @@ public class TagView: UIButton {
         setupView()
     }
     
-    private func setupView() {
+    fileprivate func setupView() {
```
Comment (xhacker) [inline TagListView/TagView.swift]: Another one.

File: TagListView/TagView.swift [hunk 4]
```diff
@@ -188,7 +188,7 @@ public class TagView: UIButton {
         return size
     }
     
-    private func updateRightInsets() {
+    fileprivate func updateRightInsets() {
```
Comment (xhacker) [inline TagListView/TagView.swift]: Another.

**Comment by migfabio** [comment]
just have a look here and make a decision about scope visibility you want to

https://github.com/apple/swift-evolution/blob/master/proposals/0025-scoped-access-level.md

**Comment by migfabio** [comment]
@xhacker scopes updated

**Comment by xhacker** [comment]
Thank you!

**Pull request merged**